src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnCache.java

```diff
@@ -293,4 +293,8 @@ private void report(Event evt) {
       cmdlineReporter.handle(evt);
     }
   }
+
+  private boolean isRemoteCache(RemoteOptions options) {
+    return options.diskCache != null && !options.diskCache.isEmpty();
```

Comment:
I think we need to flip the logic here and on line 118:

`return options.diskCache == null || options.diskCache.isEmpty();`

and 

`&& **!** isRemoteCache(options))`

Reply:
Yeah just noticed that :sweat: my bad. Just updated.

I went with `!(options.diskCache != null && !options.diskCache.isEmpty())`